lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
-
 def palindrome_permutation?(string)
+  hash = Hash.new
+  count = 0
+
+  array = string.split("")
+
+  if array.length == 0
+    return true
+  end
+
+  array.each do |letter|
+    hash[letter] = array.count(letter)
+  end
```

Comment:
Because this `.count` method loops through the entire array this section is O(n^2)!

Consider:
```suggestion
  array.each do |letter|
    if hash[letter]
      hash[letter] += 1
    else
      hash[letter] = 1
    end
  end
```